webapp/utils/__init__.py

```diff
@@ -1,6 +1,43 @@
+import base64
+import json
+from typing import Any
+
+import jwt
+import requests
+
 from webapp.exceptions import ECSTaskDoesNotExistError
 from webapp.utils.aws import CloudWatchLogsClient, ECSClient
 
+REGION = "us-east-1"
+
+
+def parse_oidc_data(
```

Comment:
I think this utility function would benefit from a docstring.

Reply:
Will add a docstring (wip).

Reply:
[Docstring has been added](https://github.com/MITLibraries/alma-sapinvoices-ui/pull/60/files#diff-20ff21c90d95c84f47465f8287c3e0b6524b04e0fffad8e50a452f9f32e52cdfR19-R29)!